app/src/common/shared/org/mozilla/vrbrowser/PermissionDelegate.java

```diff
@@ -1,8 +1,11 @@
 package org.mozilla.vrbrowser;
 
+import android.Manifest;
 import android.app.Activity;
 import android.content.Context;
 import android.content.pm.PackageManager;
+import android.support.annotation.NonNull;
+import android.support.v4.content.ContextCompat;
```

Comment:
I don't think that we need to use android.support.vr because our min API is v24.